integration-test/src/step_definitions/support/api_config_client.js

```diff
@@ -6,7 +6,7 @@ const api_config_host = process.env.api_config_host;
 function apiConfigHealthCheck() {    
     return get(api_config_host + `/info`, {
         headers: {
-            "Ocp-Apim-Subscription-Key": process.env.PAYMENTS_SUBSCRIPTION_KEY
+            "Ocp-Apim-Subscription-Key": process.env.APICONFIG_SUBSCRIPTION_KEY
```

Comment:
try to use in `common.js` this approach https://github.com/pagopa/pagopa-afm-calculator/blob/main/integration-test/src/step_definitions/support/common.js#L3

Reply:
try it for future needs